packages/ui/src/components/va-select/VaSelect.demo.vue

```diff
@@ -11,6 +11,109 @@
       />
       <p>Value: {{ defaultSingleSelect.value }}</p>
     </VbCard>
+    <VbCard
+      title="Autocomplete"
+      style="width: 400px;"
+    >
+      <va-select
+        v-model="defaultSingleSelect.value"
+        class="mb-6"
+        :options="defaultSingleSelect.options"
+        autocomplete
+      />
+      <p>Value: {{ defaultSingleSelect.value }}</p>
+    </VbCard>
+    <VbCard
+      title="Multiple autocomplete"
+      style="width: 400px;"
+    >
+      <va-select
+        v-model="defaultMultiSelect.value"
+        class="mb-6"
+        :options="defaultMultiSelect.options"
+        multiple
+        autocomplete
+      />
+      <p>Value: {{ defaultMultiSelect.value }}</p>
+    </VbCard>
+    <VbCard
+      title="Multiple autocomplete with max visible options"
+      style="width: 400px;"
+    >
+      <va-select
+        v-model="defaultMultiSelect.value"
+        class="mb-6"
+        :options="defaultMultiSelect.options"
+        :max-visible-options="2"
+        multiple
+        autocomplete
+      />
+      <p>Value: {{ defaultMultiSelect.value }}</p>
+    </VbCard>
+    <VbCard
```

Comment:
In autocomplete with new option - I can press Enter 2 times - then 2 items with same name are being created

<img width="396" alt="image" src="https://user-images.githubusercontent.com/5394573/218586640-a1bf4ad8-a1a0-4836-a286-abc8455097fd.png">

Reply:
It's base VaSelect behaviour for non-unique `allow-create` mode. But i'll handle it for autocomplete.